src/main/scala/in/rcard/http4s/tutorial/movie/MovieApp.scala

```diff
@@ -86,33 +88,33 @@ object MovieApp {
     }
   }
 
-  def directorRoutes[F[_] : Sync]: HttpRoutes[F] = {
+  def directorRoutes[F[_] : Async]: HttpRoutes[F] = {
```

Comment:
This is caused by the fact that most calls need a Sync instance, but jsonOf requires a Concurrent, which is unrelated to Sync.

The only way I could find to reconcile this conflict is to require an Async instance, which derives from both.